prover/server/publish_prover.sh

```
@@ -0,0 +1,19 @@
+# 2.0.23
+
+PROJECT_ID=$(gcloud config get-value project)
+REGION=europe-west1
+REPO_NAME=light
+TAG="${1:-latest}"  # Usage: ./publish_prover.sh [version]
+
+docker buildx build --platform linux/amd64,linux/arm64 -f Dockerfile.light -t prover-light:$TAG --load .
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)